README.md

```diff
@@ -95,31 +95,24 @@ Example usage:
 lecli get events 12345678-aaaa-bbbb-1234-1234cb123456 -f 1465370400 -t 1465370500
 lecli get events 12345678-aaaa-bbbb-1234-1234cb123456 --datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59'
 lecli get events 12345678-aaaa-bbbb-1234-1234cb123456 -r 'yesterday'
-lecli get events --loggroup myloggroup --datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59'
-lecli get events --lognick mynicknamedlog --datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59'
-lecli get events --lognick mynicknamedlog -r 'last 3 weeks'
+lecli get events --logset 12345678-aaaa-bbbb-1234-1234cb123457 --datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59'
```

Comment:
I don't think we refer to logset as uuids in the ui

Reply:
see aboce comment

Reply:
@udangel-r7 what do you mean?

Reply:
Anything in the UI/documentation typically refers to logset as names. In insightops I think it is currently impossible (?) To get a logset uuid, the same case is I think in le with redisgns etc. 

I think we should at least explain how to get a logset key or explain that this is a unique key to identify a logset

Reply:
@udangel-r7 
I reviewed the issue on logset uuid with Eoin - we want to stay with the use of logset uuid; but you were right about user access to the logset uuid as it is missing from the new webpage design - this is an oversight that Eoin is raising stories on to correct.